### DISCUSSION TITLE
Help/Error: The execution of all serverless function is marked as Errored.

### DISCUSSION THREAD

**Discussion by Vilijan** [vercel/community#289] [Help] [answered]
Hey, 

I am new to the Vercel community, so I hope I am asking the question in the right place.

I have deployed a NextJs application using Vercel, which was working seamlessly for a couple of weeks. However, all of a sudden starting from yesterday all of my serverless functions are marked as **Errored** when I explore the Usage of my app. See print screen below. 
As a serverless functions, I use only the getStaticProps with revalidation on every 60 seconds.

The following things are strange to me:
1. Everything is reported as Errored but I can't see any errors when exploring the function logs. 
2. The website is working as expected, the pages are regenerated every 60 seconds as specified. So, I am not experiencing any strange behavior on the UI part.

My question is how can I debug those Errored functions?

Any help is much appreciated, thank you.

![image](https://user-images.githubusercontent.com/22685802/152122384-56de31cd-6c69-49ce-9d25-3b6c07aa319b.png)

**Comment by carldegs** [comment]
Just saw your post and I'm also experiencing this starting Feb 1 6PM PHT (+8). 
![image](https://user-images.githubusercontent.com/14056264/152209716-bf8f9ad9-2646-4c82-9eec-55f1d44ac270.png)

**Reply by Vilijan** [reply]
Oh, wow. I feel a little bit relieved now.

**Comment by threefeetcat** [comment]
We're also seeing the same issue. But the errored rate is different, about 2/3 errored and 1/3 completed.

**Comment by troymcginnis** [comment]
We're also seeing the same issue but only on one of our projects 🤔
<img width="1020" alt="Screen Shot 2022-02-01 at 8 29 04 PM" src="https://user-images.githubusercontent.com/4299551/152415578-4d4e4d9f-a827-46d7-8d34-8c6eba45a5ac.png">

**Comment by corygibbons** [comment, marked as answer]
Experiencing the same issue on a few projects however a couple of them have seemed to resolve themselves over the last few hours. Hopefully that persists. 🤞

<img width="995" alt="Screen Shot 2022-02-03 at 17 59 00@2x" src="https://user-images.githubusercontent.com/957488/152460154-3794fea1-f701-486d-8a79-2144db0e986a.png">

**Reply by Vilijan** [reply]
I am back to normal again. It seems that the bug was fixed.

![image](https://user-images.githubusercontent.com/22685802/152492153-43035030-881e-40d1-a6e6-497bc7478548.png)
